src/Symfony/Component/Yaml/Inline.php

```diff
@@ -581,6 +584,19 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
```

Comment:
@nicolas-grekas Working on it I think we should throw an exception when the constant is not defined even if `self::$exceptionOnInvalidType` is false, what do you think **@symfony/deciders**?